### ISSUE TITLE
Should recognize already downloaded songs 

### ISSUE THREAD

**Issue by Riqium** [insaiyancvk/music_downloader#10]
When the process fails, it does not detects already downloaded songs automatically and begins to download them again in case that you want to continue/retry the process.

**Comment by insaiyancvk** [comment]
Hey, @Riqium I've released patch 0.1.0.1 (f5719252e3727884ce384a60825b18b20e78396d) where it continues the download even if one of them causes an error. 

> a way to achieve a "continous" download would be to remember the quantity of dowloaded songs, and then, skip that number of downloads.

I've already tried it and it doesn't seem possible from how I coded it. So when you download a playlist from youtube, it formats the video name and kinda makes it look "clean". So if I try to check if that name already exists in the directory, it doesn't match!
I'll give it another shot and see if I can get it fixed.
